server/apps/distributed-app/docs/modules/ROOT/pages/configure/ssl.adoc

```diff
@@ -84,6 +84,29 @@ Please note `JKS` keystore format is also supported (default value if no keystor
 </tls>
 ....
 
+When you enable TLS, you may also configure the server to request a client certificate for authentication:
```

Comment:
```suggestion
=== Client authentication via certificates

When you enable TLS, you may also configure the server to request a client certificate for authentication:
```

We might like to have a dedicated (sub) section dedicated to client authentication...

Reply:
Sure, I will update the documentation accordingly.